python/pyspark/sql/streaming.py

```diff
@@ -953,6 +953,33 @@ def csv(self, path, schema=None, sep=None, encoding=None, quote=None, escape=Non
         else:
             raise TypeError("path can be only a single string")
 
+    def table(self, tableName):
+        """Define a Streaming DataFrame on a Table and returns the result as a :class:`DataFrame`.
```

Comment:
I think it's fine to just copy and paste Scaladoc, and document the return type as below according to numpydoc style

```
Define a Streaming DataFrame on a Table. The DataSource corresponding to the table should
support streaming mode.

Parameters
...

Returns
--------
:class:`DataFrame`

```

Reply:
I saw functions here have their own description so I thought we prefer re-describing it. OK I'll copy the description. Thanks.

Reply:
Just to make sure, could you please guide about the content of doc in this method? I'm not sure whether you're suggesting to delete something or just rearrange and make a sync with Scala doc.

I'll follow the same with other method once you provide the example. Thanks!

Reply:
Ohh, I was clear. I meant, for example,

```python
"""Define a Streaming DataFrame on a Table. The DataSource corresponding to the table should
support streaming mode.

.. versionadded:: 3.2.0

Parameters
----------
tableName : str
    string, for the name of the table.

Returns
--------
:class:`DataFrame`

Notes
-----
This API is evolving.

Examples
--------
>>> csv_sdf = spark.readStream.table(input_table_name)
>>> csv_sdf.isStreaming
True
"""
```